pkg/util/status.go

```diff
@@ -38,9 +42,22 @@ func IsFailed(status apiv1.JobStatus) bool {
 	return hasCondition(status, apiv1.JobFailed)
 }
 
+func IsRunning(status apiv1.JobStatus) bool {
+	return hasCondition(status, apiv1.JobRunning)
+}
+
+func IsSuspend(status apiv1.JobStatus) bool {
+	return hasCondition(status, apiv1.JobSuspended)
```

Comment:
`hasCondition`  name now is misleading and it looks to be doing the same thing as `apimachinery/pkg/api/meta.IsStatusConditionTrue`

Reply:
Unfortunately, we use `condition.status` is typed `corev1.status`. 

https://github.com/kubeflow/training-operator/blob/a3a297213ada205dcf786e3eff4272a4b9343de1/pkg/apis/kubeflow.org/v1/common_types.go#L111

So `apimachinery/pkg/api/meta.IsStatusConditionTrue` doesn't work :(

Reply:
you could still rename the function to `isConditionTrue`

Reply:
Ah, I misunderstood @trasc 's comment. 
I will replace all functions with `IsStatusConditionTrue`. Thanks!

Reply:
Done.